docker-compose.yml

```
@@ -29,6 +29,9 @@ services:
 
             ENABLED_LINKED_FILE_TYPES: 'url,project_file'
 
+            # Enables Thumbnail generation using ImageMagick
+            ENABLE_IMAGE_MAGICK_CONVERSIONS: 'true'
```

```suggestion
            ENABLE_IMAGE_MAGICK_CONVERSIONS: 'false'
```

As per https://github.com/overleaf/docker-image/pull/115#discussion_r323149618

---

Will be enabled by default for the time being.